### DISCUSSION TITLE
How do you get the github pro pack for university students?

### DISCUSSION THREAD

**Discussion by shenalpigi** [community/community#68990] [Discussions]
### Select Topic Area

Question

### Body

I am a student and i would like to have a pro pack for my github account.

**Comment by supuna97** [comment]
Join this. https://education.github.com/

**Reply by shenalpigi** [reply]
thank you

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
